OpenMTRDemo/Forms/ExpandedImageDialog.cs

```diff
@@ -3,46 +3,67 @@
 using OpenCvSharp;
 using OpenMTRDemo.Forms;
 
-namespace OpenMTRDemo
+namespace OpenMTRDemo.Forms
 {
-    public partial class MainForm : BaseForm
+    public partial class ExpandedImageDialog : BaseForm
     {
-        OpenFileDialog OpenBrowser;
-        SaveFileDialog SaveBrowser;
-        OpenMTR.Meter meter;
+        public Mat Source, Image;
 
-        public MainForm()
+        public ExpandedImageDialog(Mat source, Mat image)
         {
-            OpenBrowser = new OpenFileDialog();
-            SaveBrowser = new SaveFileDialog();
-            OpenBrowser.Filter = "Image Files|*.bmp;*.gif;*.jpg;*.jpeg;*.png";
-            SaveBrowser.Filter = "JPG|*.jpg";
+            this.Source = source;
+            this.Image = image;
             InitializeComponent();
```

Comment:
It will be a good idea to remove any menu strip components that are unnecessary here. IE: Views and subitems

Reply:
Okay, but they're also unnecessary in the TiledFiltersForm, should I remove them in them here or in the BaseForm?